packages/swapper/src/swappers/thorchain/ThorchainSwapper.ts

```diff
@@ -65,12 +68,21 @@ export class ThorchainSwapper implements Swapper<ChainId> {
     [KnownChainIds.ThorchainMainnet]: true,
   }
 
-  private supportedSellAssetIds: AssetId[] = []
-  private supportedBuyAssetIds: AssetId[] = []
+  private supportedSellAssetIds: AssetId[] = [thorchainAssetId]
+  private supportedBuyAssetIds: AssetId[] = [thorchainAssetId]
+
   deps: ThorchainSwapperDeps
+  daemonUrl: string
+  midgardUrl: string
+  adapterManager: ChainAdapterManager
+  web3: Web3
 
   constructor(deps: ThorchainSwapperDeps) {
     this.deps = deps
+    this.daemonUrl = deps.daemonUrl
+    this.midgardUrl = deps.midgardUrl
+    this.adapterManager = deps.adapterManager
+    this.web3 = deps.web3
```

Comment:
Do we need to inject this? Doesn't seem used anywhere

Reply:
not yet, just setting the stage to use class variables and pull in helper functions to improve readability and maintainability of these swappers

Reply:
i can remove until used I suppose, just want to start cleaning these up some, but didn't want to add too much to this pr.